open-craft/xblock-eoc-journal
# [MCKIN-8137] Update for new social stats API output.

In Ficus the API output of the user social stats endpoint has changed. This PR adapts the code to work with the new data shape.

**Test instructions**:

1. Add an EOC Journal block to your course. Enable 'Display User Metrics' in block's studio edit view.
1. Before checking out the code from this branch, log into the course in apros and create a few new threads in the forums.
2. Go to the EOC block and see that it says 0 for user engagement score. All other stats except cohort average are also 0.
3. Go to apros progress page and verify that the stats look correct (are not all 0).
4. Check out this branch and restart the LMS.
5. Verify that the values in the EOC block now equal the values displayed on apros progress page.

**Reviewers**:
